scripts/effects/effect-helpers.js

```diff
@@ -140,4 +140,64 @@ export default class EffectHelpers {
 
     return activeEffect.update({ statuses: statusesArray });
   }
+
+  /**
+   * Migrates the description of Custom Convenient Effects to use the ActiveEffect5e#description field instead of the legacy flags.
+   *
+   * @param {string} id - the ID of the custom effects item
+   * @returns {Promise} a promise that resolves when all the Item's ActiveEffects descriptions have migrated
+   */
+  async _migrateCustomEffectsItemEffectDescriptions(id, migration) {
+    const item = this._settings._findCustomEffectsItem(id);
+    const updates = [];
+    const scope = Constants.MODULE_ID;
+    const key_hasmigrated = Constants.FLAGS.HAS_MIGRATED;
+    for (const e of item.effects) {
```

Comment:
Let's rename `e` to `effect` for clarity

Reply:
Done